doc/flame/structure.md

```diff
@@ -39,5 +39,10 @@ flutter:
 ```
 
 If you want to change this structure, this is possible by using the `prefix` parameter and creating
-your instances of `AssetsCache`, `ImagesCache`, `AudioCache`, and `SoundPool`s, instead of using the
+your instances of `AssetsCache`, `Images`, `AudioCache`, and `SoundPool`s, instead of using the
```

Comment:
Not your change, but `SoundPool` here should be `AudioPool` right? :thinking:

Reply:
I had the same doubt, but since I wasn't sure, I just left as is hahah, but I believe so, lets change?

Reply:
Let's do eeet!

Reply:
Or just remove it? Since there isn't any static `AudioPool` in the same sense as the others? :thinking:

Reply:
Agreed!

```suggestion
your instances of `AssetsCache`, `Images`,  and `AudioCache`, instead of using the
```